cachi2/core/config.py

```diff
@@ -22,6 +25,18 @@ class Config(BaseModel, extra="forbid"):
     requests_timeout: int = 300
     concurrency_limit: int = 5
 
+    @model_validator(mode="before")
```

Comment:
I wondered why this couldn't have been a field validator since that too supports `mode="before"`. Here's the answer: https://docs.pydantic.dev/latest/concepts/validators/#validation-of-default-values - the handling would be weird and even if it technically were `"before"` you'd already get the truthy value in the method, so there goes...